schema/adminschema/v1.json

```diff
@@ -100,6 +100,37 @@
           "$ref": "Client"
         }
       }
+    },
+    "Connector": {
+      "id": "Connector",
+      "type": "any",
+      "description": "A configuration object which describes how dex will federate identity. Because the actual structure differs for each kind of connector, the object here is unspecified."
+    },
+    "ConnectorsSetRequest": {
+      "id": "ConnectorsSetRequest",
+      "type": "object",
+      "description": "A request to set all the connectors in the dex database.",
+      "properties": {
+        "connectors": {
+          "type": "array",
+          "items": {
+            "$ref": "Connector"
+          }
+        }
+      }
+    },
+    "ConnectorsGetResponse": {
```

Comment:
Maybe it'll be clearer when I see the rest of the commit, but it seems like there should be 1 response: it either set them all or failed to do any.